osu.Game/Rulesets/UI/StandardHealthDisplay.cs

```diff
@@ -64,6 +93,21 @@ public StandardHealthDisplay()
             };
         }
 
-        protected override void SetHealth(float value) => fill.ScaleTo(new Vector2(value, 1), 200, EasingTypes.OutQuint);
+        public void BindProcessor(ScoreProcessor processor)

```

Comment:
YUP I originally had something like this, not sure why I decided to go down this route >_<